css/signup.css

```diff
@@ -0,0 +1,58 @@
+/* Colors not decided yet, will use variables later */
+.login-wrapper {
+    background-color: aliceblue;
+    margin: auto;
+    margin-top: 25vh;
+    width: 20rem;
+    box-shadow: 0px 0px 10px 1px rgb(20, 113, 194);
```

Comment:
try to avoid giving color to box shadow (suggested by vikram I think).

Reply:
Will have to think of something else here. Thanks